projects/plugins/protect/src/js/components/admin-page/index.jsx

```diff
@@ -91,21 +91,21 @@ const InterstitialPage = ( { run, hasCheckoutStarted } ) => {
 	);
 };
 
-const useCredentialState = () => {
-	const { checkCredentialsState } = useDispatch( STORE_ID );
-	const credentialState = useSelect( select => select( STORE_ID ).getCredentialState() );
+const useCredentials = () => {
+	const { checkCredentials } = useDispatch( STORE_ID );
+	const credentials = useSelect( select => select( STORE_ID ).getCredentials() );
 
 	useEffect( () => {
-		if ( ! credentialState.state ) {
-			checkCredentialsState();
+		if ( ! credentials ) {
```

Comment:
```suggestion
		if ( ! credentials.length ) {
```
An empty array will evaluate as `true` in JavaScript, so we should check the length instead.